docs/supported_apis/dataframe_supported.rst

```diff
@@ -1,676 +1,45 @@
 ``pd.DataFrame`` supported APIs
 ===================================
 
-The following table lists both implemented and not implemented methods. If you have need
-of an operation that is listed as not implemented, feel free to open an issue on the
-`GitHub repository`_, or give a thumbs up to already created issues. Contributions are
-also welcome!
+The ``pd.DataFrame`` supported APIs table lists both implemented and not implemented methods.
```

Comment:
As of now we do not list both implemented and not implemented methods. We only say about a degree of support for a parameter. Thus, this paragraph should be rephrased considering that.

Reply:
Actually we can list fully implemeted method in the table (in this case we can leave execution cell empty). Not sure that understood you correctly, what do you mean?